app/integrations/slack.server.ts

```diff
@@ -0,0 +1,106 @@
+import { json } from "@remix-run/server-runtime";
+
+import type { ZodSchema } from "zod";
+
+import type { ClientArgs } from "~/integrations/types";
+import { requireEnvVariable } from "~/utils/env";
+import { getDomainUrl } from "~/utils/misc";
+
+type SubmitParams<T extends object> = {
+  header: string;
+  meta?: Record<string, string>;
+  schema?: ZodSchema<T>;
+};
+
+export function slackClient({ request, context }: ClientArgs) {
+  const referer = request.headers.get("Referer");
+  const path = referer ? new URL(referer).pathname : "?";
+
+  const sharedMeta = {
+    UserAgent: request.headers.get("User-Agent") ?? "?",
```

Comment:
Denne er strengt tatt ikke så nyttig på sikt, men tenkte at vi kanskje kan bruke den til å se om vi kan gjøre enkle grep for å stenge ute bots, om de fremdeles dukker opp, hva tenker dere andre? 🤔